apps/tunes/views.py

```diff
@@ -63,16 +63,16 @@ def filter_queryset(self, queryset):
 
         # Try to use upvotes for this emotion and context to generate attributes for songs to return
         if self.cleaned_data.get('context'):
-            votes = self.request.user.usersongvote_set.filter(
-                emotion__name=self.cleaned_data['emotion'],
-                context=self.cleaned_data['context'],
-                vote=True
+            context_vote_data = UserSongVote.get_attributes_for_emotion_and_context(
+                self.request.user.id,
+                self.cleaned_data['emotion'],
+                self.cleaned_data['context']
```

Comment:
Can we go back to using `average` to find the attributes to use in the query for the browse playlist? It would look like:

```
votes = self.request.user.usersongvote_set.filter(
    emotion__name=self.cleaned_data['emotion'],
    context=self.cleaned_data['context'],
    vote=True
)
if votes.exists():
        attributes = average(votes, 'song__valence', 'song__energy', 'song__danceability')
        valence = attributes ['song__valence__avg']
        energy = attributes ['song__energy__avg']
        danceability = attributes ['song__danceability__avg']
```